src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/signin/DetachedSignInPage.java

```diff
@@ -0,0 +1,86 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin;
+
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.modalwindows.FacebookSignupModalComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.DetachedWindow;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FacebookAuthContext;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword.AttachedForgotPasswordPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword.DetachedForgotPasswordPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword.ForgotPasswordPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register.RegisterPage;
+
+public class DetachedSignInPage extends DetachedWindow implements SignInPage, FacebookAuthContext {
+
+  private AttachedSignInPage signInPage;
+  private final static String TITLE = "Sign in";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)